demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -587,16 +594,23 @@ export class DemoMeetingApp
       this.log('priority-downlink-policy-preset is changed: ' + presetDropDown.value);
     });
 
+    const replicaMeetingInput = document.getElementById('replica-meeting-input');
+    replicaMeetingInput.addEventListener('change', async _e => {
+      (document.getElementById('primary-meeting-external-id') as HTMLInputElement).value = "";
+    });
+
     document.getElementById('form-authenticate').addEventListener('submit', e => {
       e.preventDefault();
       this.meeting = (document.getElementById('inputMeeting') as HTMLInputElement).value;
       this.name = (document.getElementById('inputName') as HTMLInputElement).value;
       this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
       this.enableSimulcast = (document.getElementById('simulcast') as HTMLInputElement).checked;
       this.enableEventReporting = (document.getElementById('event-reporting') as HTMLInputElement).checked;
+      this.deleteOwnAttendeeToLeave = (document.getElementById('delete-attendee') as HTMLInputElement).checked;
```

Comment:
A toggle that gives us some test coverage of delete attendee that is unlikely to be accidentally baked into a builder app/easy to remove (unlike, for example, a feature which lets you kick out attendees).